### PR TITLE
Inline the refold and tactic_mode flags for the cbn tactic.

### PR DESCRIPTION
**Pull request by ppedrot**
They were unconditionally set to true, leading to a lot of dead branches.

### CODE REVIEW

**Comment by ppedrot** [comment]
This needs an assignee, @herbelin maybe?

**Comment by ppedrot** [comment]
This is trivial and up for a taker. Let's ping @SkySkimmer just in case.

File: tactics/cbn.ml [hunk 1]
```diff
@@ -453,69 +453,57 @@ let magically_constant_of_fixbody env sigma reference bd = function
           | None -> bd
         end
 
-let contract_cofix ?env sigma ?reference (bodynum,(names,types,bodies as typedbodies)) =
+let contract_cofix ~env sigma ?reference (bodynum,(names,types,bodies as typedbodies)) =
```
Comment (SkySkimmer) [inline tactics/cbn.ml:456]: Usually we have `env` as a positional argument

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot: merge now

**Pull request merged**